src/openstudio_lib/OSDocument.cpp

```diff
@@ -1406,6 +1406,19 @@ boost::optional<BCLMeasure> OSDocument::getLocalMeasure(const std::string& uid,
   return result;
 }
 
+std::vector<BCLComponent> OSDocument::getLocalComponents() const {
+  std::vector<BCLComponent> result;
+  if (m_haveLocalBCL) {
```

Comment:
(I'm always thrown off by this ("Why would local bcl not be available") until I remember why we added it... )